pkgs/by-name/ta/tableplus/linux.nix

```
@@ -0,0 +1,69 @@
+{
+  autoPatchelfHook,
+  dpkg,
+  fetchurl,
+  gtk3,
+  gtksourceview3,
+  krb5,
+  lib,
+  libgee,
+  libsecret,
+  libxkbcommon,
+  stdenv,
+  wrapGAppsHook3,
+  xorg,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tableplus";
+  version = "0.1.284";
+
+  src = fetchurl {
+    url = "https://deb.tableplus.com/debian/22/pool/main/t/tableplus/tableplus_${finalAttrs.version}_amd64.deb";
+    hash = "sha256-TrYRRpjSVRD721XEd0gwGPv4RNq/uwaswPW2J8oVjXQ=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+    wrapGAppsHook3
+  ];
+
+  buildInputs = [
+    gtk3
+    gtksourceview3
+    krb5
+    libgee
+    libsecret
+    libxkbcommon
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXext
+    xorg.libXi
+    xorg.libXrandr
+    xorg.libXrender
+    xorg.libxcb
+  ];
+
+  unpackPhase = ''
+    dpkg-deb -x $src .
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp -r opt/tableplus $out/
+    ln -s $out/tableplus/tableplus $out/bin/tableplus
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Database management made easy";
+    homepage = "https://tableplus.com";
+    license = lib.licenses.unfree;
+    maintainers = with lib.maintainers; [ rhydianjenkins ];
```

See https://github.com/NixOS/nixpkgs/blob/18b698b6564e86fa05be8668f43a7ca28ee076c4/pkgs/README.md#meta-attributes
```suggestion
    maintainers = with lib.maintainers; [ rhydianjenkins ];
    mainProgram = "tableplus";
```

---

Seem to be missing `mainProgram` and `sourceProvenance`, from reading that link. I'll add those :+1: